faunadb/client.py

```diff
@@ -12,6 +12,34 @@
 from faunadb.request_result import RequestResult
 from faunadb._json import parse_json_or_none, to_json
 
+class _LastTxnTime(object):
+  """Wraps tracking the last transaction time supplied from the database."""
+  def __init__(self):
+    self.lock = threading.Lock()
+    self.time = None
+
+  def get_time(self):
```

Comment:
I _think_ it's more idiomatic if you use the `@propery`? Unless you specifically want to state that this is a thread safe getter. In which case, `get_` isn't a very explicit prefix for that.